src/pybind/mgr/dashboard/run-frontend-unittests.sh

```diff
@@ -45,6 +45,14 @@ fixed, please check the I18N suggestions in 'HACKING.rst':\n"
   failed=true
 fi
 
+# npm resolutions
+npm run fix:audit
+resolutions=`git status | grep package-lock.json`
+if [[ "$resolutions" ]]; then
```

Comment:
hm, I just noticed that the script uses `[ ... ]` everywhere, except in this if block and the preceding one.

Could you change it to `[ ... ]` and also make this change in the previous `if` block?

- if [[ ! -z $i18n_lint ]]; then
+ if [ "$i18n_lint" ]; then

Reply:
I usually use `[ ... ]` for simple conditionals, just to avoid bashisms when I can get away with it.

Reply:
Done.